Gaijinizer patch/Map038.txt

```diff
@@ -4,108 +4,110 @@
 
 なんとかシェルターを抜けれたな。
 > CONTEXT: Map038/events/4/pages/0/5/Dialogue/yuuma
-I thought what would happen at one time ...
-I managed to get out of the shelter.#MTLed
+For a while there, I wasn't sure how things were
+going to turn out...
+But you've managed to get out of there, huh.#Ron
 > END STRING
 
 > BEGIN STRING
 地上に出たら、市街地を抜けて、俺のいる港を目指す。
 
 その港に着いたら、一緒にこの地域を脱出しよう。
 > CONTEXT: Map038/events/4/pages/0/9/Dialogue/yuuma
-When I get out on the ground, I will go through
-the city and aim for the port where I am.
-When you arrive at the port, let's escape this
-area together.#MTLed
+Once you're above ground, head through the city
+and try to get to the port where I am.
+We'll escape together when you get to the port.#Ron
 > END STRING
```

Comment:
Let's spruce up the language here so it's not as stiff:
>_**When you get to the surface, make your way
through the city and try to get to the port where I'm at.
Once you get here, we can escape together.**_